jigasi/Dockerfile

```diff
@@ -11,7 +11,7 @@ LABEL org.opencontainers.image.documentation="https://jitsi.github.io/handbook/"
 ENV GOOGLE_APPLICATION_CREDENTIALS /config/key.json
 
 RUN apt-dpkg-wrap apt-get update && \
-    apt-dpkg-wrap apt-get install -y jigasi jq && \
+    apt-dpkg-wrap apt-get install -y jigasi jq jitsi-autoscaler-sidecar && \
```

Comment:
Let's install Node from the NodeSource repo please.

Reply:
This happens in the base-java image now.